src/hooks/useProjects.ts

```diff
@@ -1,139 +1,18 @@
 import axios from 'axios'
-import { PV_V1, PV_V2 } from 'constants/pv'
-import { V1ArchivedProjectIds } from 'constants/v1/archivedProjects'
-import { V2ArchivedProjectIds } from 'constants/v2v3/archivedProjects'
 import { BigNumber } from 'ethers'
 import { DBProject, DBProjectQueryOpts, DBProjectRow } from 'models/dbProject'
-import {
-  InfiniteSGQueryOpts,
-  SGEntity,
-  SGEntityKey,
-  SGQueryOpts,
-  SGWhereArg,
-} from 'models/graph'
 import { Json } from 'models/json'
-import { ProjectState } from 'models/projectVisibility'
-import { PV } from 'models/pv'
-import { Project } from 'models/subgraph-entities/vX/project'
-import { V1TerminalVersion } from 'models/v1/terminals'
 import { useMemo } from 'react'
 import {
   UseInfiniteQueryOptions,
   UseQueryOptions,
   useInfiniteQuery,
   useQuery,
 } from 'react-query'
-import { getSubgraphIdForProject, parseSubgraphEntity } from 'utils/graph'
 import { formatQueryParams } from 'utils/queryParams'
-import { parseDBProjectJson, parseDBProjectsRow } from 'utils/sgDbProjects'
-
-import useSubgraphQuery from './useSubgraphQuery'
-
-interface ProjectsOptions {
-  pageNumber?: number
-  projectId?: number
-  projectIds?: number[]
-  orderBy?: 'createdAt' | 'currentBalance' | 'volume' | 'paymentsCount'
-  orderDirection?: 'asc' | 'desc'
-  pageSize?: number
-  state?: ProjectState
-  keys?: (keyof Project)[]
-  terminalVersion?: V1TerminalVersion
-  pv?: PV[]
-}
+import { parseDBProject } from 'utils/sgDbProjects'
 
```

Comment:
sg projects queries are now all deprecated. also no need for archived_ids here, as archived state will be stored in the DB and automatically filtered via the API endpoint